httpserver-buffer.lua

```diff
@@ -0,0 +1,48 @@
+-- httpserver-connection
```

Comment:
ITs not a rename but an alternative implementation of the connection overlay.
I changed sending static files to not use a coroutine as that is quite memory consuming.
So to still be able to use the header.lua I needed a buffer to collect the separate sends in a string and then send them out.
I then only store the filename and the current position to send the chunks of the file.